src/ctap2/attestation.rs

```diff
@@ -1,5 +1,6 @@
+use super::server::HMACGetSecretOutput;
```

Comment:
I think you can add `HMACGetSecretOutput` 2 lines down, where there are already an server-imports.

Reply:
Done! https://github.com/mozilla/authenticator-rs/pull/337/commits/5035146fe7a0c6ae4282fbae5c92bca2c0742879